### ISSUE TITLE
What is surface quality here? - "Surface is different" option improvement

### ISSUE THREAD

**Issue by CTJoriginal** [Helium314/SCEE#500]
**Problem**
There is this annoying pop-up that asks you if the surface is correct when you want to change the incorrect surface on surface quality quest:

[Context.jpg](https://github.com/Helium314/SCEE/assets/48687492/e15cf917-a59b-4aca-bdd1-ae397a5fdc19)

I think this is a completely unnecessary step that could be removed, as you see what the surface is on the quest screen already.

**Proposed Solution**
Instead of screening users with this frustrating pop-up, we just skip that all together. Then, to avoid unnecessary changesets, we could check if the user chooses the same or different surface than road in quest had before change. 

In a nutshell:

Surface is different -> Pick correct surface -> Check if surface has actually changed -> return to preverious quest with new surface

This check step could also be skipped altogether.
